packages/x-babel-config/jest.js

```diff
@@ -1,4 +1,6 @@
 const getBabelConfig = require('./');
 const babelJest = require('babel-jest');
 
-module.exports = babelJest.createTransformer(getBabelConfig());
+const base = getBabelConfig([{ node: 6 }], 'commonjs');
```

Comment:
with the change to an object, this would be:

```suggestion
const base = getBabelConfig({
	targets: [{ node: 'current' }],
	modules: 'commonjs'
});
```

you can also use `current` as the target node version, so `babel-preset-env` only transforms what it needs for the version of node it's running in.